content/collections/advanced-techniques/en/flag-prerequisites.md

```diff
@@ -71,33 +72,33 @@ Flag prerequisites are extremely flexible and have many use cases. That said, he
 
 ### Release groups
 
-Use flag prerequisites to build a primary feature with multiple sub-features. Sub-features require the primary feature to be `on`, unless a user is individually included as a tester in one of the sub-features. Targeting and bucketing applied to the primary feature are effectively applied to all sub-features which list the primary feature as a prerequisite.
+Use flag prerequisites to build a primary feature with multiple sub-features. Sub-features require the primary feature to be on, unless a user is individually included as a tester in one of the sub-features. Targeting and bucketing applied to the primary feature are also applied to all sub-features that list the primary feature as a prerequisite.
 
-Common use cases for release groups are:
+Common use cases for release groups include:
 
 - Actively developing large feature releases with many developers and teams.
 - Provisioning users to primary SKUs with add-ons.
-- Simplifying feature flag logic in code.
+- Simplifying feature flag logic in the code.
 
 ![Diagram of example release group.](/docs/output/img/experiment/release-group.drawio.svg)
 
-This example contains a `primary-feature` flag and `sub-feature` flags which list the primary feature as a prerequisite.
+This example contains a primary-feature flag and sub-feature flags that list the primary feature as a prerequisite.
 
-The `primary-feature` flag targets all users where user property `premium` is `true` with 100% allocation. Therefore, sub-features only evaluate if the user has the required user property, and meet the sub-feature's criteria -- unless the user is individually included in the sub-feature's testers section.
+The primary-feature flag targets all users where the user property premium is true with 100% allocation. Therefore, sub-features are only evaluated if the user has the required user property and meets the sub-feature's criteria—unless the user is individually included in the sub-feature's testers section.
```

Comment:
code markup was useful here. we usually write property names and values this way.